src/chatClient.cpp

```diff
@@ -2293,8 +2374,13 @@ bool GroupChatRoom::syncWithApi(const mega::MegaTextChat& chat)
     }
     else
     {
-        clearTitle();
-        KR_LOG_DEBUG("Empty title received for group chat %s", Id(mChatid).toString().c_str());
+        // With this contidion some unecesary title changed notification is avoided.
```

Comment:
Please, check spelling of `contidion` and `unecesary`

Reply:
--> With this condition, some unnecessary notifications about title-changes are avoided.